backend/build.gradle

```diff
@@ -37,7 +37,7 @@ subprojects {
     /* default dependencies for all projects*/
     dependencies {
         implementation 'org.apache.logging.log4j:log4j-core:2.20.0'
-        implementation 'org.apache.logging.log4j:log4j-slf4j2-impl:2.20.0'
+        compileOnly 'org.apache.logging.log4j:log4j-slf4j2-impl:2.20.0'
```

Comment:
While this change is not needed anymore, this might be more correct if we want to do something like asserting log messages at some point:
https://github.com/molgenis/molgenis-emx2/blob/360b6599095da22d2184d01682fc30d78afb33f7/backend/build.gradle#L74-L75